wolfssl/wolfcrypt/ecc.h

```diff
@@ -391,6 +391,10 @@ struct ecc_key {
     ecc_context_t ctx;
 #endif
 
+#ifdef WOLFSSL_ECDSA_SET_K
+    mp_int sign_k;
```

Comment:
What about a public key for verify?  Would a pointer be better here to minimize resource use?

Reply:
This is not the private key but the random k value used in signing.
Enabling setting of K to allow known answer tests when signing with ECDSA.

Changed to a pointer as it is a temporary item.

Reply:
Sure, but ecc_key can be used in the context of a public key for verify only.  And then the user would have to pay for the resource of sign_k.

Reply:
sign_k is now a pointer in last commit.